index.js

```diff
@@ -103,8 +103,8 @@ module.exports = {
       'components/dialog/dialog-theme.scss'
     ];
 
-    var pathBase = this.project.addonPackages['ember-paper'].path;
-    var angularMaterialPath =  'node_modules/angular-material-source/src';
+    var pathBase = this.project.nodeModulesPath;
```

Comment:
In npm 3 there is a common `node_modules` folder. In npm 2.x each dep has its own `node_modules`.
With this change, we make sure it always points to the correct node_modules. Is this correct?